pkg/operator/controllers/suite_test.go

```diff
@@ -72,6 +74,14 @@ var _ = BeforeSuite(func() {
 		ErrorIfCRDPathMissing: true,
 	}
 
+	//If config is passed to a constructor, whatever fields constructor uses need to be defined explicitly
+	//This does not pull values from operator.yaml like it does in main.go
+	//For suite_test the env defaults should be used, defined in const above
+	config := &Config{
```

Comment:
while this works, the Config struct defines some default values and it would be great to test them as well. Any chance we can use the same code we have in main.go? Like

```golang
cfg := &Config{}
if err := env.Parse(cfg); err != nil {
    log.Fatal(err, "Cannot load configuration from environment variables")
}
```

Reply:
We might need to provide some dummy values for the ones that should not be empty, like THUNDERNETES_INIT_CONTAINER_IMAGE and THUNDERNETES_INIT_CONTAINER_IMAGE_WIN

Reply:
Latest commit uses the parse from main.go and filled in some defaults.